src/App.js

```diff
@@ -1,10 +1,10 @@
 import React, { Fragment } from 'react';
 import { BrowserRouter as Router, Route } from "react-router-dom";
 import Chart from "./containers/Chart";
-import { NavMenu } from "./components/NavMenu";
 import './App.scss';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import 'antd/dist/antd.css';
+import { NavMenu } from "./components/NavMenu";
 
```

Comment:
imports splitted group order:
1 - reacts dependencies and other native ones
2 - your constants, services, components
3 - your styles

```
import React, { Fragment } from 'react';
import { BrowserRouter as Router, Route } from "react-router-dom';
import 'bootstrap/dist/css/bootstrap.min.css';
import 'antd/dist/antd.css'

import Chart from './containers/Chart";
import { NavMenu } from './components/NavMenu";

import './App.scss';
```